tests/ui/tests/README.md

```diff
@@ -0,0 +1,66 @@
+# Tests
+
+## Overview
+
+This project contains API Gateway integration tests for Kyma Dashboard.
+
+## Prerequisites
+
+If you want to use an existing cluster, you must copy your cluster's kubeconfig file to `fixtures/kubeconfig.yaml`.
+
+## Installation
+
+To install the dependencies, run the `npm clean install` command.
+
+## Test development using Headless mode with Chrome browser
```

Comment:
```suggestion
## Test Development Using Headless Mode with Chrome Browser
```
We've changed the guidelines to use title case for headings. I updated Istio docs but haven't updated API Gateway yet